tests/integration_tests/performance/test_process_startup_time.py

```diff
@@ -22,6 +24,19 @@ def test_startup_time_new_pid_ns(test_microvm_with_api):
     microvm.jailer.new_pid_ns = True
     _test_startup_time(microvm)
 
+    pid_file_path = "{}/{}".format(microvm.jailer.chroot_path(),
+                                   FC_PID_FILE_NAME)
+    assert os.path.exists(pid_file_path)
+
+    # Read the PID stored inside the file.
+    with open(pid_file_path) as file:
+        fc_pid = int(file.readline())
+    file.close()
+
+    # We need to explicitly kill the Firecracker pid, since it's different from
+    # the jailer pid which gets killed on object destruction.
+    utils.run_cmd(f'kill -9 {fc_pid}', ignore_return_code=True)
```

Comment:
Should we just add this in the standard microvm destructor?

Look for pidfile, if found `kill -9 $(cat $pidfile)`.

Would cover future tests as well.

Reply:
good idea. done